modules/govuk/manifests/apps/collections.pp

```diff
@@ -82,5 +86,8 @@
     "${title}-FEATURE-FLAG-ACCOUNTS":
         varname => 'FEATURE_FLAG_ACCOUNTS',
         value   => $feature_flag_accounts_var;
+    "${title}-MEMCACHE_SERVERS":
+        varname => 'MEMCACHE_SERVERS',
```

Comment:
Where is this variable used?  I don't see it referenced in collections.

Reply:
It's used in Dalli (which is our memcached client):

https://github.com/petergoldstein/dalli/blob/1fbef3caf060d7d4e852c2a38e7c2987e208f0c7/lib/dalli/client.rb#L35

We should probably mention that in a comment though.

Reply:
It's the place the Ruby memcache driver looks for it: https://github.com/petergoldstein/dalli/blob/1fbef3caf060d7d4e852c2a38e7c2987e208f0c7/lib/rack/session/dalli.rb#L27

Reply:
I've added a comment explaining this, which hopefully helps.